### ISSUE TITLE
Complete programmatic access

### ISSUE THREAD

**Issue by jemhuntr** [krasimir/gifffer#29]
Hi! Nice little library you've got here.

Would love to use this, although instead of using it as a user-interactive gif player, I would love to use it for programmatically controlling GIFs. Is it possible to:

1: Completely disable toggling of the GIF play state via clicks.
2: Programmatically `.play()`, `.pause()`, `.rewind()`, and `.stop()` GIF playback.

This allows for autonomous control of the GIFs, allowing me to use any events as trigger to the playback. For example, this allows me to make a GIF animated logo appear static on the header of my app, playing and pausing the animation playback based on global loading state. Or perhaps, I would be able to make it such that GIFs play on-hover instead of on-click.

**Comment by krasimir** [comment]
Actually wha the library does is replacing a screenshot of the gif with the actual gif. So there is no frame-by-frame animation. Which means that we can't control the timeline and can't stop, pause.

**Comment by jemhuntr** [comment]
Right. Perhaps we can make it such that we can `.attach()` and `.detach()` the actual GIF instead? And an option to disable the click events plus the play icon.
